src/components/AppHeader.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <div class='app-header'>
+    <div class='ui secondary pointing menu'>
+      <router-link to='/' class='item'>Feed</router-link>
```

Comment:
What if their latest post is of a different content type than `text`? I like where you are going with this, just wanted to see what your thoughts were on this aspect.